README.md

```diff
@@ -313,21 +313,27 @@ flutter_native_splash:
   web: false
 ```
 
-Great, now comes the fun part running the new command!
-
-The new command is:
+If you'd like to generate only a single flavor (maybe you are
+testing something out), you can use only the single command like this:
 
 ```bash
 # If you have a flavor called production you would do this:
 flutter pub run flutter_native_splash:create --flavor production
 
 # For a flavor with a name staging you would provide it's name like so:
-flutter pub run flutter_native_splash:create --flavor staging
+flutter pub run flutter_native_splash:create --flavor acceptance
```

Comment:
I think this should not have been changed from staging to acceptance?
Also `flutter pub run` is now deprecated.  User `dart run` instead.

Reply:
Regarding the `staging` to `acceptance`, it was actually a mistake to write staging because all of the examples above have the name of acceptance. The YAML files, the image files and the rest of the text mentions acceptance not staging. I actually made a mistake here when I wrote it initially. 😄 

Could you please share some link to where it states that the `flutter pub run` is deprecated? I'm unable to find anything official. 🤔

Reply:
I was not able to find anything official either, but when I run `flutter pub run`, it outputs the following message:
```
Deprecated. Use `dart run` instead.
```

Reply:
@jonbhanson Yeah I saw that, it might be that it was unintentionally left there in preparation to actually deprecate it.
As per your request, I will change `flutter pub run` to `dart run`